Intersect (Core)/Config/PlayerOptions.cs

```diff
@@ -5,26 +5,26 @@
     /// </summary>
     public partial class PlayerOptions
     {
-        /// <summary>
-        /// Intersect default for initial player bank slots
-        /// </summary>
-        public const int DefaultInitialBankSlots = 100;
-
         /// <summary>
         /// Unlinks the timers for combat and movement to facilitate complex combat (e.g. kiting)
         /// </summary>
         public bool AllowCombatMovement { get; set; } = true;
 
         /// <summary>
-        /// Enables the client feature 'Auto-turn to target'.
+        /// When enabled, 'Auto-turn to target' is limited to entities within the front view of players.
         /// </summary>
-        public bool EnableAutoTurnToTarget { get; set; } = true;
+        public bool AutoTurnToTargetAccountDirection { get; set; } = false;
```

Comment:
This is not a new feature, this is just a boolean to limit a feature that already exists, which behavior was updated to work along with diagonal movement and directions, this is because of some users, including @blinkuz, who want to be able to target things with ease but, they are cornerned about the fact that the `auto-turn to target` feat allows the player to turn around towards target behind them. I'll rename it to `LimitAutoTurnToTarget` and resort alphabetically the player options, that being said, if you still consider that this feature limitation shouldn't be included at all on this PR, i can purge it out from it and work towards it in another one for sure.